### ISSUE TITLE
'Segment/SEGAnalytics.h' file not found when using version 6.1.0

### ISSUE THREAD

**Issue by renamari** [AppsFlyerSDK/segment-appsflyer-ios#36]
We rely heavily on Segment integrations and this AppsFlyer integration is exactly what we need for our project. Our Podfile includes the following:

```
pod 'Analytics',                    '4.1.1', :inhibit_warnings => true
pod 'Segment-Adjust',               '3.1.0', :inhibit_warnings => true
pod 'Segment-Firebase',             '2.7.0', :inhibit_warnings => true
pod 'segment-appsflyer-ios',        '6.1.0', :inhibit_warnings => true
```

However, the Appsflyer iOS integration is the only one having issues building, and fails with the following error:

`#import <Segment/SEGAnalytics.h>               // 'Segment/SEGAnalytics.h' file not found`

Other integrations such as Adjust and Firebase contain the following imports and make me think this would resolve the issues we're seeing with the AppsFlyer integration:

```
#if defined(__has_include) && __has_include(<Analytics/SEGAnalytics.h>)
#import <Analytics/SEGAnalytics.h>
#else
#import <Segment/SEGAnalytics.h>
#endif
```

However if you think there may be something I'm missing, please let me know! Thanks so much.

**Comment by af-margot** [comment]
Hi @renamari ,

Thank you for pointing this out! 
I see that you have Analytics 4.1.1, which should work with **Segment/SEGAnalytics.h** as they changed the names in v4.1.0. Not sure why it's not working. Anyway, we will check your suggestion, see if we can add it, and come back to you!

**Comment by mlazari** [comment]
Seeing the same issue with `Analytics 4.1.1` and `segment-appsflyer-ios 6.1.0`:
![Screen Shot 2020-11-09 at 16 05 09](https://user-images.githubusercontent.com/4928274/98551063-8c13fb00-22a5-11eb-8cf0-2e234a24fd2f.png)

**Comment by mlazari** [comment]
@af-margot Looking in the [CHANGELOG for 4.1.0](https://github.com/segmentio/analytics-ios/blob/master/CHANGELOG.md#version-410-19-october-2020) the [PR for that change](https://github.com/segmentio/analytics-ios/pull/942/files) is named "Swift Rename". I don't know Swift, but could that rename be Swift only? I also see that conditional import in other integrations we use. And looks like there is an open PR for this repo too that adds that: https://github.com/AppsFlyerSDK/segment-appsflyer-ios/pull/37/files

**Comment by af-margot** [comment]
Hi @mlazari,

Yes, we merged the PR and we will release a new version with it.

**Comment by af-margot** [comment]
Hi @mlazari , @renamari 

We released [v6.0.4 ](https://github.com/AppsFlyerSDK/segment-appsflyer-ios/releases/tag/6.0.4)with the PR. It should be available soon.

**Comment by mlazari** [comment]
Thank you @af-margot
